apiserver/gui_test.go

```diff
@@ -43,21 +43,28 @@ type guiSuite struct {
 var _ = gc.Suite(&guiSuite{})
 
 // guiURL returns the complete URL where the Juju GUI can be found, including
-// the given hash and pathAndquery.
-func (s *guiSuite) guiURL(hash, pathAndquery string) string {
-	return s.urlFromBase(apiserver.GUIURLPathPrefix, "", pathAndquery)
+// the given hash and pathAndQuery.
+func (s *guiSuite) guiURL(hash, pathAndQuery string) string {
+	if pathAndQuery == "" {
+		return s.URL(apiserver.GUIURLPathPrefix, nil).String()
+	}
+	prefix := apiserver.GUIURLPathPrefix
+	if strings.HasPrefix(pathAndQuery, "static/") || pathAndQuery == "config.js" {
```

Comment:
We also needed to update the url generation for the tests to not put the /dashboard/ prefix on the static or config.js queries.